courses/models.py

```diff
@@ -392,6 +392,7 @@ class CourseRun(TimestampedModel):
     )
 
     live = models.BooleanField(default=False)
+    self_paced_certificates = models.BooleanField(default=False)
```

Comment:
Nit: I think this should probably be renamed a bit - a) boolean fields should be prefixed with either `is_` or `has_` b) the certificates being self paced sounds a bit odd, is it rather that the course run is self paced?

Reply:
Atleast for now the self paces would only make sense for certificates but you're right if we make it generic it would make more sense for other conditions that we might place based on self paced of a course run. I've changed it to `is_self_paced`